src/pages/sns/faq.tsx

```diff
@@ -741,6 +808,44 @@ function SnsFaqPage() {
                 </Link>
               </p>
             </Faq>
+            
+            <Faq title="What is voting power and how do I get it?">
+              SNS DAOs have a stake-based governance system. This means the more
+              tokens participants stake in their neurons, the more voting power
+              their neurons have. In addition to the stake, a neuron's voting power
+              is dependent on the{" "}
+              <Link href="https://support.dfinity.org/hc/en-us/articles/4404298574612-What-is-dissolve-delay-#:~:text=The%20dissolve%20delay%20is%20a,or%20lock%20up%20your%20ICP.">
+                  dissolve delay
+              </Link> bonus and age bonus. Each SNS DAO
+              decides these bonuses in its parameters.
+            </Faq>
+            
+            <Faq title="What is neuron following?">
+              <p>
+                Following neurons is a way to delegate votes both on the NNS and in SNSs.
+                In this case, each SNS neuron can either manually vote on proposals or
+                follow another neuron’s voting decisions. Neurons are followed based on
+                various topics, such as making SNS treasury transfers or upgrading the dapp.
+                Neurons can be set to follow other neurons on specific topics or to follow
+                other neurons on ‘All Topics’ – a catch-all is applied for the proposal
```

Comment:
```suggestion
                other neurons on ‘All Topics’ – a catch-all which is applied for the proposal
```
(not sure, but it read a bit weird to me)